aggregator/options.go

```diff
@@ -656,9 +686,15 @@ func (o *options) computeFullGaugePrefix() {
 
 // By default we use e.g. ".p50", ".p95", ".p99" for the 50th/95th/99th percentile.
 func defaultTimerQuantileSuffixFn(quantile float64) []byte {
-	s := fmt.Sprintf(".p%0.0f", quantile*10000)
-	// Trim 2 rightmost "0"s.
-	return []byte(strings.TrimSuffix(strings.TrimSuffix(s, "0"), "0"))
+	s := fmt.Sprintf("%0.0f", quantile/defaultQuantilePrecision)
+	return []byte(fmt.Sprintf(".p%s", trimQuantileSuffix(s)))
+}
+
+func trimQuantileSuffix(s string) string {
+	for strings.HasSuffix(s, "0") && len(s) > 2 {
```

Comment:
hm I don't think this works for say, the 5th percentile.

Perhaps use something like https://play.golang.org/p/VXrfQm8A9p?